judgments/tests/factories.py

```diff
@@ -1,11 +1,10 @@
 from typing import Any
 from unittest.mock import Mock
 
-from caselawclient.models.judgments import Judgment
+from caselawclient.models.documents import Document
 
 
-class JudgmentFactory:
-    # "name_of_attribute": ("name of incoming param", "default value")
+class DocumentFactory:
```

Comment:
When replacing this I noticed that the factory had `judgment_date_as_string` which we still haven't decided on what the analogous for a press summary would be.

Similarly `has_supplementary_materials` might not be appropriate for a press summary. Potentially having a general document object with all the generic properties makes sense, but then some other objects which link them together such as.


Decision -> JudgmentDocument (Inherits Document) , Press Summary (Inherits Document)

A decision wouldnt have a uri but could be stored in django/postgres maybe? It could hold the ncn, whether it has supplementary materials, decision_date_as_string etc. ?

And eventually (once we sort case numbers and case pages etc. ): Case -> Decision 1, Decision 2, Decision 3....

Similar to what @timcowlishaw suggested the other day?

Im sure there are wholes in my particualr example but I think it would be good to sit down and have a proper think about the way we want to relate them a bit more.

Reply:
Yeah, there are definitely _lots_ of cases where we need to figure out if a behaviour is inherent to all documents, or if it's class-specific. It might be that `judgment_date` and `document_publication_date` are different things we care about... or it might not.

For now I think we just plough on and we can fix up if needed.

Reply:
Yeah agreed for here, but its worth us finalising the metadata ticket to have clear AC for what metadata we want to expose on a press summary and be able to edit in the EUI.